### DISCUSSION TITLE
Why did I have to install bootstrap on my computer?

### DISCUSSION THREAD

**Discussion by HolisticDev1992** [twbs/bootstrap#38239] [Q&A] [answered]
Don’t I just download the files and go from there? Why did I have to get NPM and all the other dependencies installed on my computer to use a file only built in languages that my browser understands??

**Comment by bkpecho** [comment, marked as answer]
You don't have to. You can link Bootstrap at the top of a page for style. NPM, as a package manager, collects these frameworks on your local machine for when it transpile the code into the finished product that renders in the browser.
